tests/memory/index.test.js

```diff
@@ -115,7 +115,7 @@ describe("Memory tests", () => {
     10 * 60 * 1000,
   );
 
-  it(
+  it.only(
```

Comment:
It would fail as Vitest does not allow `it.only` in CI because it would skip some tests that were intended to be run

Reply:
Yes forgot that one :/ thanks